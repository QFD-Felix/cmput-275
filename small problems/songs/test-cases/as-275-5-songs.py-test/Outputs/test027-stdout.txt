14 30 4 5 48 8 34 2 11 6 10 0 14 25 9 12 2 42 44 22 15 1 8 11 0 20 2 9 1 3 40 64 56 0 17 30 3 5 43 24 3 25 7 1 59 2 0 7 0 1 29 2 21 15 4 0 0 0 3 1 7 3 7 22 12 9 18 37 3 3 7 51 19 27 1 3 7 39 2 7 51 6 0 74 31 14 4 10 45 0 2 5 1 18 0 32 10 4 0 3 7 4 11 21 44 37 2 7 4 0 0 10 8 6 20 24 14 0 0 37 12 8 24 0 1 1 23 1 25 6 59 6 30 5 42 28 0 3 39 4 4 0 3 16 7 9 6 0 22 11 0 57 21 2 10 33 2 59 27 2 21 6 4 2 4 7 53 2 11 6 14 0 57 1 9 3 6 3 45 22 35 17 9 9 2 5 1 17 14 21 17 6 1 2 0 62 0 30 2 0 9 2 10 2 7 15 0 10 10 13 11 2 2 25 53 9 10 16 37 12 15 7 21 14 6 0 18 20 14 30 2 34 4 1 6 17 3 11 12 19 1 5 18 2 27 11 5 6 1 13 14 3 29 13 29 22 12 6 35 2 12 12 16 7 25 4 42 37 6 21 17 5 3 13 21 13 3 3 4 0 5 11 16 13 0 16 16 0 3 0 4 0 0 12 0 2 3 10 0 1 3 0 0 0 0 0 0 0 0 0
