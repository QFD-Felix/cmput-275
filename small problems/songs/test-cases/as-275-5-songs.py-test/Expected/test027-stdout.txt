14 11 4 5 43 8 34 2 11 6 10 0 14 18 9 12 2 44 46 22 15 1 8 2 0 20 2 9 1 3 43 67 6 0 0 18 3 5 46 27 3 28 6 1 62 4 0 9 6 2 0 1 30 2 14 16 4 0 0 0 3 1 8 2 8 23 1 12 9 7 37 3 3 7 51 19 27 1 3 7 11 2 7 51 6 0 76 0 14 4 10 45 0 2 5 1 18 0 32 10 4 0 3 7 4 11 21 44 37 2 7 4 0 0 10 4 2 20 24 3 0 0 1 10 8 24 0 1 1 23 1 25 6 28 1 25 4 44 30 0 3 0 4 4 0 3 18 7 9 6 0 24 13 0 5 23 2 12 35 4 53 6 32 27 2 21 6 4 2 4 7 53 2 11 6 14 0 57 1 3 3 6 3 5 22 35 17 9 9 2 5 1 17 14 21 17 6 1 2 0 62 0 30 2 0 9 2 10 2 7 15 0 10 8 13 11 2 2 25 53 9 10 1 37 12 15 2 21 14 4 0 18 20 14 1 2 28 1 1 2 17 3 3 12 19 1 5 18 2 27 11 5 1 1 4 14 3 31 13 8 22 12 6 37 2 5 12 16 7 27 4 6 43 6 23 19 5 3 8 25 15 3 3 6 19 7 1 6 2 18 15 0 7 20 25 3 6 4 0 0 10 0 4 0 0 3 12 4 1 5 0 1 0 3 0 0 0 0 0 0 0
